specification.markdown

```diff
@@ -404,23 +404,25 @@ PathMaxOccurrence     = 1*DIGIT ; no upper limit
 
 The `*`, `(` and `)` symbols are interpreted as defined in [@!RFC5234].
 
-The EBMLPathAtom part of the EBMLElementPath MUST be equal to the name attribute of the EBML Schema.
+The EBMLAtomName part of the EBMLLastPath MUST be equal to the `@name` attribute of the EBML Schema.
 
 The starting PathDelimiter of the path corresponds to the root of the EBML Document.
 
-In some cases the EBMLLastParent part of the path is an EBMLGlobalParent. A path with a EBMLGlobalParent defines a Global Element; see (#global-elements). Any path that starts with the EBMLFixedParent of the Global Element and matches the occurrences found in the GlobalParentOccurence is a valid path for the Global Element. 
+If the EBMLLastPath is an EBMLPathAtomRecursive, the EBML Element can occur within itself recursively (see (#recursive)).
```

Comment:
I'd suggest that this sentence should immediately follow the one on line 407, such as

>The EBMLAtomName part of the EBMLLastPath MUST be equal to the `@name` attribute of the EBML Schema. If the EBMLLastPath is an EBMLPathAtomRecursive, the EBML Element can occur within itself recursively (see (#recursive)).

Reply:
OK